chaincode/tuple.go

```diff
@@ -30,6 +30,7 @@ const (
 	StatusFailed   = "failed"
 	StatusDone     = "done"
 	StatusCanceled = "canceled"
+	StatusAborted  = "canceled"
```

Comment:
maybe add a comment here to explain why is it equal to canceled, for now, that could be helpful...